docs/pages/application-access/okta/usersync.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Okta User Synchronization
+description: Managing Okta users in Teleport
+---
+
+## User Synchronization
```

Comment:
I think it's a good idea to have a page that explains synchronization process. But I propose we make it about Okta SCIM integration in general rather than just users so @mdwn can add info about Groups sync here as well, and we have a central place where we can point people who want to learn about our SCIM integration to. Maybe call it `Okta Synchronization & SCIM`?

Also, I would consider adding a section to our FAQ page, something along the lines of "Does Teleport support SCIM?" and linking to this page from there as well.

Reply:
Seems reasonable. The only thing that possibly worries me about it is that the amount of syncing happening in the Okta service is a lot -- it could be overwhelming to read about.

Reply:
If the page is structured well and have ToC then I don't think it would be an issue.

Reply:
Should this merge with the "Architecture" page, then? That seems to have the most information about how resources are synced. Or do we think there is enough content about how Okta resources are mapped to Teleport resources to warrant its own page, outside the mechanics of the process?

Reply:
@tcsc Architecture page is for low(-ish)-level implementation details, I don't think we should merge them. IMO there should be a page in the documentation that explains to users what actually happens to Okta user and groups and how SCIM fits into all of this. I think this page is a good candidate for that.